src/crate/client/sqlalchemy/tests/function_test.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; -*-
+#
+# Licensed to CRATE Technology GmbH ("Crate") under one or more contributor
+# license agreements.  See the NOTICE file distributed with this work for
+# additional information regarding copyright ownership.  Crate licenses
+# this file to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.  You may
+# obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# However, if you have executed another commercial license agreement
+# with Crate these terms will supersede the license and you may use the
+# software solely pursuant to the terms of the relevant commercial agreement.
+
+from unittest import TestCase
+from unittest.mock import MagicMock
+
+import sqlalchemy as sa
+from sqlalchemy.sql.sqltypes import TIMESTAMP
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import Session
+
+fake_cursor = MagicMock(name="fake_cursor")
+
+
+class DialectTest(TestCase):
```

Comment:
This test doe not seem to be executed:

```
$ bin/test --list-tests | grep test_date_trunc_type_is_timestamp
```

Reply:
Good shout - I ran the test manually, but forgot to add it to the zope test suite. Will fix.